sdk/react/ooyalaSkinCore.js

```diff
@@ -133,7 +133,24 @@ OoyalaSkinCore.prototype.shouldShowLandscape = function() {
 };
 
 OoyalaSkinCore.prototype.shouldShowDiscoveryEndscreen = function() {
-  return (this.skin.state.upNextDismissed == true && this.skin.props.endScreen.screenToShowOnEnd == "discovery");
+  // Only care if discovery on endScreen should be shown
+  if (this.skin.props.endScreen.screenToShowOnEnd !== "discovery") {
```

Comment:
Do we populate this config with default values if, for example, `endScreen` is not provided?  I'm not sure if this would be an issue on the PBA side, but on the html5 skin we started using a safe get function so that this wouldn't crash if the skin config was incomplete.